src/pretty.rs

```diff
@@ -0,0 +1,68 @@
+use partiql_value::Value;
+
+use std::fmt::Result;
+use std::fmt::Write;
+
+pub trait PrettyPrint {
```

Comment:
can the [`TODO`](https://github.com/partiql/partiql-rust-cli/blob/35ffacfd7c6b83c6bbff356971094d29b2650f17/src/pretty.rs#L6) be added?

```suggestion
// TODO move to `partiql-lang-rust/partiql-value`
pub trait PrettyPrint {
```